bazel/repositories.bzl

```diff
@@ -7,6 +7,11 @@ load(":genrule_repository.bzl", "genrule_repository")
 load(":patched_http_archive.bzl", "patched_http_archive")
 load(":repository_locations.bzl", "REPOSITORY_LOCATIONS")
 load(":target_recipes.bzl", "TARGET_RECIPES")
+load("@bazel_tools//tools/cpp:windows_cc_configure.bzl",
```

Comment:
I think it's fine. `windows_cc_configure.bzl` does expose `find_vc_path`, `setup_vc_env_vars`, `find_msvc_tool`.
I believe they are quite common functions that're needed when setting up MSVC related repo. The function interface didn't change since the beginning, so I think it's stable enough to let others to depend on it.

Reply:
The question is why do we need to setup the MSVC environment variables? @sesmith177

Reply:
The dependencies that are built with scripts (e.g. https://github.com/envoyproxy/envoy/blob/75909a7d16f0291ccd0bbe29517ac387a34701ea/ci/build_container/build_recipes/cares.sh) need to have the MSVC environment variables set. So we set the variables up when calling the `repositories.bat` script that ends up building them